### PR TITLE
Update dependencies

### LINKED ISSUE #5709: Update jackson-databind libraries

### PR DESCRIPTION
**Pull request by anuraaga**
Fixes #5709

### CODE REVIEW

File: licenses/licenses.md [hunk 1]
```diff
@@ -1,146 +1,221 @@
 
 #javaagent
 ##Dependency License Report
-_2021-11-23 20:07:26 PST_
+_2022-04-08 05:12:23 UTC_
```
Comment (trask) [inline licenses/licenses.md:4]: 👍

File: licenses/jackson-jr-objects-2.13.2.jar/META-INF/LICENSE [hunk 2]
```diff
@@ -0,0 +1,8 @@
+This copy of Jackson JSON processor streaming parser/generator is licensed under the
+Apache (Software) License, version 2.0 ("the License").
+See the License for details about distribution rights, and the
+specific rights regarding derivate works.
```
Comment (anuraaga) [inline licenses/jackson-jr-objects-2.13.2.jar/META-INF/LICENSE:4]: @trask I couldn't find a simple way to exclude `licenses` from spellcheck

https://github.com/client9/misspell

Any thoughts?

Comment (trask) [reply licenses/jackson-jr-objects-2.13.2.jar/META-INF/LICENSE:4]: https://github.com/client9/misspell#how-do-you-check-an-entire-folder-recursively

> The following examples selects all `.txt` files that are *not* in the `vendor` directory:
>
> ```bash
> find . -type f -name '*.txt' | grep -v vendor/ | xargs misspell -error
> ```

I can mess around with it tomorrow

**Review by trask — APPROVED** [review]

File: licenses/licenses.md [hunk 3]
```diff
@@ -1,146 +1,221 @@
...
+**34** **Group:** `io.opentelemetry` **Name:** `opentelemetry-sdk-trace` **Version:** `1.12.0` 
+> - **POM Project URL**: [https://github.com/open-telemetry/opentelemetry-java](https://github.com/open-telemetry/opentelemetry-java)
+> - **POM License**: Apache License, Version 2.0 - [http://www.apache.org/licenses/LICENSE-2.0](http://www.apache.org/licenses/LICENSE-2.0)
+
+**35** **Group:** `io.opentelemetry` **Name:** `opentelemetry-semconv` **Version:** `1.12.0-alpha` 
 > - **POM Project URL**: [https://github.com/open-telemetry/opentelemetry-java](https://github.com/open-telemetry/opentelemetry-java)
 > - **POM License**: Apache License, Version 2.0 - [https://www.apache.org/licenses/LICENSE-2.0](https://www.apache.org/licenses/LICENSE-2.0)
 
-**28** **Group:** `net.bytebuddy` **Name:** `byte-buddy-dep` **Version:** `1.11.22` 
+**36** **Group:** `io.prometheus` **Name:** `simpleclient` **Version:** `0.14.1`
```
Comment (mateuszrzeszutek) [inline licenses/licenses.md:156]: Are we still including prometheus deps?

**Review by mateuszrzeszutek — APPROVED** [review]

File: licenses/licenses.md (hunk 3 above)
Comment (anuraaga) [reply licenses/licenses.md:156]: This is probably still getting added transitively through SDK until it's released

**Pull request merged**